core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -953,7 +953,7 @@ protected function generateButtons($arrRow, $strTable, $arrRootIds=array(), $bln
 					if (($params['act'] ?? null) == 'toggle' && isset($params['field']))
 					{
 						// Hide the toggle icon if the user does not have access to the field
-						if (!System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_EDIT_FIELD_OF_TABLE, $this->strTable . '::' . $params['field']))
+						if (($GLOBALS['TL_DCA'][$strTable]['fields'][$params['field']]['toggle'] ?? false) !== true || !System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_EDIT_FIELD_OF_TABLE, $strTable . '::' . $params['field']))
```

Comment:
Isn't this a BC break? Because the condition will also be true if the field has no `toggle` property, which is the case in Contao <4.13. So all toggles will disappear until the developers have added `'toggle' => true` in their DCAs.

Reply:
there was no `act=toggle` in previous versions

Reply:
Apparently there was, because there is a `($params['act'] ?? null) == 'toggle'` in line 953 and you did not add it in this PR.

Reply:
this PR **improves** the toggle operation added in Contao 4.13. I think you have too many PRs on your plate 😂🤗

Reply:
This is very possible. 😄